src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -570,7 +571,11 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
                 role="alert"
                 aria-live="assertive"
               >
-                {saveError}
+                <ModalMessage
+                  variant={VARIANT.NEGATIVE}
+                >
+                  {saveError}
+                </ModalMessage>
```

Comment:
Based on the mockup, I think we actually want the `ModalMessage` to be in the middle of the footer. The `ErrorSection` and `ErrorMessage` layouts were things I had added back in f34f41e before we had a direction for the modal message, so those can probably be removed entirely and we can just stick the `LoadSpinner` and the `ModalMessage` in the footer. Also, we can probably remove the `showError` prop from the `MeetingModalBodyGrid` just stick with a static grid -- that should make our lives easier with other modals as well.

A full patch for my suggestions:

```diff
diff --git a/src/client/components/pages/Courses/MeetingModal.tsx b/src/client/components/pages/Courses/MeetingModal.tsx
index db99f70b..c425922f 100644
--- a/src/client/components/pages/Courses/MeetingModal.tsx
+++ b/src/client/components/pages/Courses/MeetingModal.tsx
@@ -66,18 +66,15 @@ interface MeetingModalProps {
 /**
  * Utility component to style content within meeting modal body
  */
-const MeetingModalBodyGrid = styled.div<{showError: boolean}>`
+const MeetingModalBodyGrid = styled.div`
   width: 75vw;
   height: 75vh;
   display: grid;
   grid-template-areas:
     "meet room"
     "note room"
-    "err err"
   ;
-  grid-template-rows: ${({ showError }) => (showError
-    ? 'auto 1fr min-content'
-    : 'auto 1fr 0')};
+  grid-template-rows: 'auto 1fr';
   grid-template-columns: 1fr 1fr;
   grid-column-gap: ${fromTheme('ws', 'xlarge')};
 `;
@@ -136,22 +133,6 @@ const NotesSection = styled.div`
   grid-area: note;
 `;
 
-const ErrorSection = styled.div`
-  grid-area: err;
-  display: flex;
-  flex-direction: column;
-  justify-content: end;
-  align-items: center;
-`;
-
-const ErrorMessage = styled.p`
-  font-family: ${fromTheme('font', 'bold', 'family')};
-  font-size: ${fromTheme('font', 'bold', 'size')};
-  font-weight: ${fromTheme('font', 'bold', 'weight')};
-  color: ${fromTheme('color', 'text', 'negative')};
-  text-align: center;
-`;
-
 const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
   isVisible,
   onClose,
@@ -514,7 +495,7 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
         {`Meetings for ${catalogNumber} - ${term} ${calendarYear}`}
       </ModalHeader>
       <ModalBody>
-        <MeetingModalBodyGrid showError={!!saveError || saving}>
+        <MeetingModalBodyGrid>
           <MeetingScheduler>
             <MeetingSchedulerHeader>{`Meeting times for ${catalogNumber}`}</MeetingSchedulerHeader>
             <MeetingSchedulerBody>
@@ -564,21 +545,6 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
               />
             </RoomAvailabilityBody>
           </RoomAvailability>
-          <ErrorSection>
-            {saving && <LoadSpinner>Saving Meetings</LoadSpinner>}
-            {!!saveError && (
-              <ErrorMessage
-                role="alert"
-                aria-live="assertive"
-              >
-                <ModalMessage
-                  variant={VARIANT.NEGATIVE}
-                >
-                  {saveError}
-                </ModalMessage>
-              </ErrorMessage>
-            )}
-          </ErrorSection>
         </MeetingModalBodyGrid>
       </ModalBody>
       <ModalFooter>
@@ -589,6 +555,14 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
         >
           Save
         </Button>
+        {saving && <LoadSpinner>Saving Meetings</LoadSpinner>}
+        {!!saveError && (
+          <ModalMessage
+            variant={VARIANT.NEGATIVE}
+          >
+            {saveError}
+          </ModalMessage>
+        )}
         <Button
           onClick={onModalClose}
           variant={VARIANT.SECONDARY}
```